docs/reference/filters/money/index.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: default
+title: Money
+parent: Filters
+---
+
+When using a `money` filter you have access to the following attributes.
+
+# money
+
+The provided integer returned as a formatted string including the registered currency.
+
+##### input
+{% raw %}
+```liquid
+  <h1>{{ 150 | money }}</h1>
+```
+{% endraw %}
+
+##### output
+Assuming the registered currency is GBP
+
+{% raw %}
+```html
+  <h1>£1.50</h1>
+```
+{% endraw %}
+
+# apply_promotion
+
+The discounted [Price]({% link docs/reference/objects/product/price.md %}) of a provided [Price]({% link docs/reference/objects/product/price.md %}) with the provided [Promotion]({% link docs/reference/objects/product/promotion.md %}) applied.
```

Comment:
How about the following:

> This filter will return the discounted version of a given [price]({% link docs/reference/objects/product/price.md %}) when a [promotion]({% link docs/reference/objects/product/promotion.md %}) is applied. It will return a [price]({% link docs/reference/objects/product/price.md %}) object.